test/_fixtures/error-payload.json

```diff
@@ -0,0 +1,12 @@
+{
+    "fullDisplayName": "node-test-commit-arm #3087",
+    "result": "FAILURE",
+    "changeSet": {
+        "items": [
+            {
+                "commitId": "7190bd23810f0880d95605bf6f785224a345d477"
+            }
+        ]
+    },
```

Comment:
Mind explaining the rational behind this?

I can't really see the use of anything other than one commit sha